ios/Runner/SecureChannelHandler.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SPDX-License-Identifier: BSD-2-Clause-Patent
+//  Copyright © 2022 Bitmark. All rights reserved.
+//  Use of this source code is governed by the BSD-2-Clause Plus Patent License
+//  that can be found in the LICENSE file.
+//
+
+import Foundation
+import Flutter
+import Combine
+
+class SecureChannelHandler: NSObject {
+
+    static let shared = SecureChannelHandler()
+    static var shouldShowSplash = true
```

Comment:
maybe use can make it non-static and use `SecureChannelHandler.shared.shouldShowSplash` to check?